pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: dart_doc_syncer
-version: 1.0.1
+publish_to: none
```

Comment:
I had added back a version number because it helped me know which version I was using under site-webdev. Is there any way we can have the `version` and `publish_to` fields? /cc @kwalrath

Reply:
Actually, I think so...

Reply:
\o/